overlord/fdestate/backend/reseal_sb_test.go

```diff
@@ -1,4 +1,5 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
+//go:build !nosecboot
```

Comment:
hm why doesn't this come up on Arch/Fedora/openSUSE?

Reply:
I have not done Fedora yet. As for the rest? Perhaps snapd.mk?

Reply:
snapd.mk is only passing nosecboot and nooptee. I think the problem was with the debian packaging which is actively removing files from the tree to work around dh-golang silly ness